metrics-core-api/src/main/java/com/alibaba/metrics/CachedGauge.java

```diff
@@ -48,23 +49,24 @@ protected CachedGauge(Clock clock, long timeout, TimeUnit timeoutUnit) {
     public T getValue() {
         if (shouldLoad()) {
             this.value = loadValue();
+            lastUpdate = System.currentTimeMillis();
         }
         return value;
     }
 
     @Override
     public long lastUpdateTime() {
-        return reloadAt.get();
```

Comment:
maybe turn reloadAt to milliseconds directly is better

Reply:
I think there's a little bit difference between `reloadAt` and `lastUpdate`.
`reloadAt` = `lastUpdate` + `timeout`
Usually, `timeout` is a very small value, so they can be very close.